app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -129,30 +129,27 @@ def user_data_supported?
 
     private
 
+    def fog_credentials
+      { :provider => :openstack,
+        :openstack_api_key  => password,
+        :openstack_username => user,
+        :openstack_auth_url => url,
+        :openstack_tenant   => tenant,
+        :openstack_identity_endpoint => url }
```

Comment:
To be fair I don't know, I'm figuring that out. In network, volume and compute, I tried basic operations using Rails console and I didn't need to pass a different URL. I'd go with this at least for RC2 so that we don't block any tester that uses Openstack. 

@dhague , you introduced these changes in Fog, any ideas? Neither [pull request comments](https://github.com/fog/fog/pull/3582) nor the changelog indicate much.